### PR TITLE
fix: Unable to disable page view on POST page

### PR DESCRIPTION
**Pull request by yolee599**
Issue #83

### CODE REVIEW

File: layouts/_default/terms.html [hunk 1]
```diff
@@ -24,7 +24,7 @@ <h1>
         <ul class="category-list">
           {{- range $name, $items := $cats }}
           <li class="category-list-item">
-            <a class="category-list-link" href="{{ "/categories/" | relLangURL }}{{ $name | urlize | lower  }}" data-pjax-state="">{{ .Page.Title }}</a>
+            <a class="category-list-link" href="{{ "/categories/" | relLangURL }}{{ $name | urlize  }}" data-pjax-state="">{{ .Page.Title }}</a>
```
Comment (elkan1788) [inline layouts/_default/terms.html:27]: 这段代码冲突了，这里的`$name`变量最后是要进行小写，避免访问出现404。

File: layouts/_default/terms.html [hunk 2]
```diff
@@ -41,7 +41,7 @@ <h1>
       <div class="tag-cloud-tags">
         {{ $randNums := (seq 10) }}
         {{- range $name, $items := $tags }}
-        <a class="tag-cloud-{{ index (shuffle $randNums) 0 }}" href="{{ "/tags/" | relLangURL }}{{ $name | urlize | lower  }}">{{ .Page.Title }}
+        <a class="tag-cloud-{{ index (shuffle $randNums) 0 }}" href="{{ "/tags/" | relLangURL }}{{ $name | urlize  }}">{{ .Page.Title }}
```
Comment (elkan1788) [inline layouts/_default/terms.html:44]: 同上

**Review by elkan1788 — CHANGES_REQUESTED** [review]
请检查下冲突部分的代码，需要调整。

**Comment by yolee599** [comment]
The develop branch is behind the main branch, so change to hotfix.

**Comment by elkan1788** [comment]
> The develop branch is behind the main branch, so change to hotfix.

Sorry, it's not a hotfix apply, recommand to change your code to develop branch, thanks.

**Pull request closed without merging**